setup.go

```diff
@@ -45,6 +45,10 @@ func setup(c *caddy.Controller) error {
 					return err
 				}
 				s.backendTLS = t
+			case "balancer":
+				if !c.Args(&s.balancerName) {
+					return c.Errf("balancer missing argument")
+				}

```

Comment:
Does this make "balancer" a required field in the Caddyfile?

"balancer" should be optional, especially important for backward compatibility.

Reply:
I don't believe this makes it required. It just means if you specify `balancer` you need to provide an argument.